package.json

```diff
@@ -1,25 +1,31 @@
 {
   "name": "route-recognizer",
-  "version": "0.2.8",
+  "version": "0.3.0",
   "description": "A lightweight JavaScript library that matches paths against registered routes.",
   "author": "Yehuda Katz",
   "repository": {
     "type": "git",
     "url": "https://github.com/tildeio/route-recognizer.git"
   },
   "devDependencies": {
-    "broccoli-file-creator": "^1.1.1",
+    "@types/qunit": "^2.0.31",
     "broccoli-funnel": "^1.0.6",
     "broccoli-merge-trees": "^1.1.4",
     "broccoli-rollup": "^1.0.2",
+    "broccoli-string-replace": "^0.1.1",
+    "broccoli-typescript-compiler": "^1.0.1",
     "do-you-even-bench": "^1.0.2",
     "ember-cli": "^2.7.0",
     "glob": "^7.1.1",
     "jshint": "^2.9.3",
-    "rollup-plugin-replace": "^1.1.1"
+    "qunitjs": "^2.1.0",
+    "rollup-plugin-buble": "^0.14.0",
+    "tslint": "^4.0.2",
+    "typescript": "^2.1.4"
   },
   "main": "dist/route-recognizer.js",
-  "jsnext:main": "dist/es6/route-recognizer.js",
+  "jsnext:main": "dist/route-recognizer.es.js",
```

Comment:
Just makes the source map not `../route-recognizer.ts` and seems silly to make a folder for one file.